site/src/pages/TaskPage/TaskPage.tsx

```diff
@@ -126,10 +130,25 @@ const TaskPage = () => {
 	} else if (agent && ["created", "starting"].includes(agent.lifecycle_state)) {
 		content = <TaskStartingAgent agent={agent} />;
 	} else {
+		const chatApp = selectChatApp(task);
 		content = (
 			<PanelGroup autoSaveId="task" direction="horizontal">
 				<Panel defaultSize={25} minSize={20}>
-					<TaskSidebar task={task} />
+					{chatApp ? (
+						<TaskAppIFrame active task={task} app={chatApp} />
+					) : (
+						<div className="h-full flex items-center justify-center p-6 text-center">
+							<div className="flex flex-col items-center">
+								<h3 className="m-0 font-medium text-content-primary text-base">
+									Chat app not found
+								</h3>
+								<span className="text-content-secondary text-sm">
+									Please, make sure your template has a chat sidebar app
+									configured.
+								</span>
+							</div>
+						</div>
+					)}
```

Comment:
We're missing an indication of the sidebar app health status. This is important for troubleshooting. Can we add that back in?